doc/design/architecture.md

```diff
@@ -106,7 +106,7 @@ None --> Planning +------>------->------> Installing +---> Complete
 | RequiresApproval | occurs when using manual approval, will not transition phase until `approved` field is true    |
 | Installing       | resolved resources in the InstallPlan `Status` block are being created                      |
```

Comment:
Worth updating this while you're in the neighborhood -- it may still be resolving bundles, or, now, waiting for a valid OperatorGroup.